src/vs/workbench/browser/parts/activitybar/activitybarPart.ts

```diff
@@ -128,6 +128,53 @@ export class ActivitybarPart extends Part implements IActivityBarService {
 			getContextMenuActionsForComposite: () => [],
 			getDefaultCompositeId: () => this.viewletService.getDefaultViewletId(),
 			hidePart: () => this.layoutService.setSideBarHidden(true),
+			dndHandler: {
```

Comment:
It seems like we are duplicating this code in the activityBar and the panelPart.
Could we use some shared CompositeDragAndDrop to be shared between this two parts?